src/content/developers/docs/nodes-and-clients/index.md

```diff
@@ -125,7 +125,7 @@ If you're more of a technical user, dive into more details and options on how to
 
 ## Alternatives {#alternatives}
 
-Setting up your own node can cost you time and resources but you don’t always need to run your own instance. In this case, you can use a third party API provider like [Infura](https://infura.io), [Alchemy](https://alchemyapi.io), [Chainstack](https://chainstack.com), or [QuikNode](https://www.quiknode.io). Alternatively, [ArchiveNode](https://archivenode.io/) is a community-funded Archive node that hopes to bring archive data on the Ethereum blockchain to independent developers who otherwise couldn't afford it. For an overview of using these services, check out [nodes as a service](/developers/docs/nodes-and-clients/nodes-as-a-service/).
+Setting up your own node can cost you time and resources but you don’t always need to run your own instance. In this case, you can use a third party API provider like [Infura](https://infura.io), [Alchemy](https://alchemyapi.io), [Chainstack](https://chainstack.com), [Pocket](https://pokt.network/) or [QuikNode](https://www.quiknode.io). Alternatively, [ArchiveNode](https://archivenode.io/) is a community-funded Archive node that hopes to bring archive data on the Ethereum blockchain to independent developers who otherwise couldn't afford it. For an overview of using these services, check out [nodes as a service](/developers/docs/nodes-and-clients/nodes-as-a-service/).
```

Comment:
We've spoken about this before, but let's just remove this section and link to our nodes as a service page here. We can't list everyone here, and we want to remain credibly neutral. Thoughts @wackerow @jmcook1186 @corwintines?

Reply:
I'm actually one of the protocol engineers at Pocket.

I was reading this page for other reasons, and figured we could also add Pocket when I encountered this paragraph,

We're one of 6 providers in https://docs.ethers.io/v5/api/providers and the only one that's a decentralized network of node providers. Would really appreciate if you consider the change.

Reply:
> EDIT: Were you suggesting removing the whole "Alternatives" section?

Yes. We should remove alternatives entirely. Or else we have to decide who to include/not include here subjectively.

Reply:
@Olshansk I've created #8767 to cover the above. I'm removing the specific reference to Pocket in this PR in the interim and will get the rest of this merged.